alert-filters.json

```diff
@@ -1,32 +1,46 @@
 {
-  "add": [
+  “add”: [
```

Comment:
I think the double quote character is wrong

Reply:
corrected it now.